thrifty-schema/src/main/kotlin/com/microsoft/thrifty/schema/Linker.kt

```diff
@@ -446,7 +446,7 @@ internal class Linker(
                 val expectedPath = "$includeName.thrift"
                 constant = program.includes
                         .asSequence()
-                        .filter { p -> p.location.path == expectedPath } // TODO: Should this be ==, or endsWith?
+                        .filter { p -> File(p.location.path).name == expectedPath }
```

Comment:
If you feel up to it, I think it would make sense to add `val name` to `Location` itself, rather than implement it here in this lambda.

Reply:
Hi @benjamin-bader, I noticed that there is an existing API on Location for checking this. So I changed to use that instead. Please take another look.